contracts/ArtifactRegistry.sol

```diff
@@ -41,4 +45,9 @@ contract ArtifactRegistry is IArtifactRegistry, Ownable, ERC721Full {
 
     return (artwork.artist, artwork.metaUri);
   }
+
+  function logARR(address from, address to, uint256 tokenId, uint price) public {
```

Comment:
I don't think logARR is a good function name seeing as it also is in charge of transferring the token